sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/JdbcDriverTest.java

```diff
@@ -32,11 +32,17 @@
 import org.apache.beam.sdk.extensions.sql.mock.MockedBoundedTable;
 import org.apache.beam.sdk.schemas.Schema.TypeName;
 import org.apache.calcite.jdbc.CalciteConnection;
+import org.junit.Before;
 import org.junit.Test;
 
 /** Test for {@link JdbcDriver}. */
 public class JdbcDriverTest {
 
+  @Before
+  public void before() throws Exception {
+    Class.forName("org.apache.beam.sdk.extensions.sql.impl.JdbcDriver");
```

Comment:
I'd like to understand the environments that require this. What runtime do they have? This should be handled by the service file for java.sql.Driver in Java 7+. How do we test that path keeps working?

Reply:
If it varies on env, I think we'd need a test matrix. We used to have a bit but shut down our Travis job.

Reply:
TODO(me): when I am at the computers where it is broken, report my env to the JIRA.